src/globals/scss/core/_form-group.scss

```diff
@@ -0,0 +1,15 @@
+@include exports("form-group") {
+
+  .govuk-form-group {
+    @include govuk-responsive-margin($govuk-spacing-responsive-6, "bottom");
+
+    .govuk-form-group:last-of-type {
+      margin-bottom: 0; // Remove margin from last item in nested groups of form-groups
+    }
+  }
+
+  .govuk-form-group--error {
+    padding-left: $govuk-spacing-scale-3;
+    border-left: $govuk-border-width solid $govuk-red;
```

Comment:
Related to the whole border width conversation, this should probably be using either `$govuk-border-width-error` or `$govuk-border-width-error-group`, or those variables shouldn't exist and we should just be using `$govuk-border-width`?